<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dubbing Automation Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="mainTabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="autoModeTab">
       <attribute name="title">
        <string>Автоматический режим</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="templateComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="createTemplateButton">
            <property name="text">
             <string>Создать</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editTemplateButton">
            <property name="text">
             <string>Редактировать</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteTemplateButton">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Номер серии:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="episodeNumberLineEdit">
            <property name="placeholderText">
             <string>например, 11 (можно оставить пустым в ручном режиме)</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Русская аудиодорожка:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="audioPathLineEdit">
            <property name="placeholderText">
             <string>Выберите .wav или .aac файл</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="selectAudioButton">
            <property name="text">
             <string>Обзор...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Оригинальный MKV:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="mkvPathLineEdit">
            <property name="placeholderText">
             <string>Выберите .mkv файл</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="selectMkvButton">
            <property name="text">
             <string>Обзор...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0" colspan="1">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_41">
            <property name="text">
             <string>Свои субтитры:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="overrideSubsPathEdit">
            <property name="placeholderText">
             <string>Субтитры из MKV будут проигнорированы</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browseOverrideSubsButton">
            <property name="text">
             <string>Обзор...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="1" colspan="1">
         <layout class="QHBoxLayout" name="horizontalLayout_51">
          <item>
           <widget class="QLabel" name="label_44">
            <property name="text">
             <string>Свои надписи:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="overrideSignsPathEdit">
            <property name="placeholderText">
             <string>Надписи из MKV будут проигнорированы</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browseOverrideSignsButton">
            <property name="text">
             <string>Обзор...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPlainTextEdit" name="logOutput">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QLabel" name="progressLabel">
          <property name="text">
           <string>Текущий этап:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QProgressBar" name="downloadProgressBar">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="visible">
           <bool>false</bool>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="2">
         <widget class="QPushButton" name="startButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>СТАРТ</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="manualModeTab">
       <attribute name="title">
        <string>Ручная сборка</string>
       </attribute>
       <layout class="QVBoxLayout" name="manualTabLayout"/>
      </widget>
      <widget class="QWidget" name="renderTab">
       <attribute name="title">
        <string>Ручной рендер</string>
       </attribute>
       <layout class="QVBoxLayout" name="renderTabLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
